### PR TITLE
Hotfix content updates

### PR DESCRIPTION
**Pull request by StacSch**
Updates to Help, need to go on Staging for Greg
@kaitlin 
@kristine.sweet
@nmonga91

### CODE REVIEW

File: src/help/changelog.md [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
+  - [Update to Rule B5](#/help?section=B5)
+  - [Object Class Validation Update](#/help?section=objectClass)
+  - [MAX Group Permissions Allow for Different Permissions for a User Who is Part of a Multiple Agency Permission Group](#/help?section=maxGroup)
+  - [Browser Requirements & Known Issues](#/help?section=browser)
+  - [Accessibility Statement](#/help?section=accessibilityStatemen)
+
+##### Generate D Files Outside of a Submission{section=Dfiles}
 
- Users can now add additional columns to their submission files (A-C) that will be returned in their warning and error files. To use this feature, add any column to your submission data and prefix the header with "flex\_". For example, a column named "flex_reportingbureau" will be ignored for validation purposes but returned for any rows that have errors in the warnings and error reports. 
+Users will be able to generate D files outside the context of a submission. In other words, they won't need to validate A-C to be able to generate D files.
```
Comment (kaitlin) [inline src/help/changelog.md]: We should add the link here to point agencies to the page where they can do this.

File: src/help/validations.csv [hunk 2]
```diff
@@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T
...
-A,A9,"ContractAuthorityAmountTotal_CPE= CPE aggregate value for GTAS SF 133 line #1540 + #1640, for the same reporting period",Fatal Error,Implemented,Calculation
-A,A10,BorrowingAuthorityAmountTotal_CPE= aggregate value for GTAS SF 133 line #1340 + #1440 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A11,SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE= aggregate value for GTAS SF 133 line #1750 + #1850 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A12,AdjustmentsToUnobligatedBalanceBroughtForward_CPE= aggregate value for GTAS SF 133 line #1010 through 1042 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A14,GrossOutlayAmountByTAS_CPE= value for GTAS SF 133 line #3020 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A15,UnobligatedBalance_CPE= value for GTAS SF 133 line #2490 for the same reporting period.,Fatal Error,Implemented,Calculation
+A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period","Warning currently, Fatal Error in future release",Implemented,Calculation
```
Comment (kaitlin) [inline src/help/validations.csv]: it looks like this is reverting earlier changes. These should have been changed to errors but it looks like this PR is reverting it back to "Warning currently, error in future release"

**Comment by StacSch** [comment]
I’ll take care of that.

Stacia

File: src/help/validations.csv [hunk 3]
```diff
@@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T
...
 B,B4,"ObligationsDeliveredOrdersUnpaidTotal in File B = USSGL 4901 + 4981 in File B for the same date context (FYB or CPE). This applies to the program activity and object class level. Note for FYB values, only 4901 is expected to have a balance other than zero.",Warning,Implemented,Calculation
-B,B5,GrossOutlayAmountByProgramObjectClass (File B) = GrossOutlaysUndeliveredOrdersPrepaidTotal (File B) + GrossOutlaysDeliveredOrdersPaidTotal (File B) for the same date context (FYB or CPE).,Warning,Implemented,Calculation
+B,B5,GrossOutlayAmountByProgramObjectClass_FYB (File B) = GrossOutlaysUndeliveredOrdersPrepaidTotal_FYB (File B) + GrossOutlaysDeliveredOrdersPaidTotal_FYB (File B). GrossOutlayAmountByProgramObjectClass_CPE (File B) should reflect year-to-date activity as of the end of the reporting period: GrossOutlayAmountByProgramObjectClass_CPE = (GrossOutlaysUndeliveredOrdersPrepaidTotal_CPE less GrossOutlaysUndeliveredOrdersPrepaidTotal_FYB) + (GrossOutlaysDeliveredOrdersPaidTotal_CPE less GrossOutlaysDeliveredOrdersPaidTotal_FYB),Warning,Implemented,Calculation
```
Comment (kaitlin) [inline src/help/validations.csv]: I think Kristine needs to review these

**Comment by StacSch** [comment]
I need to make changes to two of the files, then put in the PRs.

Stacia

**Pull request closed without merging**